pkgs/by-name/dt/dtbloader/package.nix

```diff
@@ -0,0 +1,62 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  llvmPackages,
+  nix-update-script,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "dtbloader";
+  version = "1.2.2";
+
+  src = fetchFromGitHub {
+    owner = "TravMurav";
+    repo = "dtbloader";
+    tag = finalAttrs.version;
+    hash = "sha256-5Efxi0cojhq9mqB5VMShXx/Sp4CE6Cvr1pcwnUm5zlo=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = with llvmPackages; [
+    clang
+    lld
+    llvm
+  ];
+
+  # Disable hardening which adds incompatible flags like -fPIC
+  hardeningDisable = [ "all" ];
+
+  buildPhase = ''
+    make -j$NIX_BUILD_CORES
```

Comment:
Why is an explicit `buildPhase` necessary? If you want parallel builds, just use `enableParallelBuilding = true;`.